polkadot/node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -1270,7 +1270,7 @@ impl Initialized {
 		if import_result.has_fresh_byzantine_threshold_against() {
 			let blocks_including = self.scraper.get_blocks_including_candidate(&candidate_hash);
 			for (parent_block_number, parent_block_hash) in &blocks_including {
-				gum::trace!(
+				gum::warn!(
```

Comment:
should the log below also be warn?

Reply:
Good point, given that a revert was not possible - I would say yes.